src/aap_eda/api/serializers/activation.py

```diff
@@ -585,6 +586,12 @@ class ActivationReadSerializer(serializers.ModelSerializer):
     organization = OrganizationRefSerializer()
     rules_count = serializers.IntegerField()
     rules_fired_count = serializers.IntegerField()
+    ruleset_stats = YAMLSerializerField(
+        required=False,
+        allow_null=True,
```

Comment:
Since the model has an empty dict as default values, these two values are not necessary, am I wrong?